cli/cook/querying.py

```diff
@@ -26,21 +26,16 @@ class Clusters:
 
 def __distinct(seq):
```

Comment:
Nitpick: should this be moved to `util.py`

Reply:
Done